### ISSUE TITLE
Interpolated strings cause template render error.

### ISSUE THREAD

**Issue by KeithHenry** [frhagn/Typewriter#129]
Add a TS interpolated string in a template, for instance:

```ts
console.log(`Called ctor for class $Name with ${parameterName}`);
```

Causes the template to not render with the error:

> 07:35:12.310 WARNING: Template error: CS1519 Invalid token '}' in class, struct, or interface member declaration 

Removing the `${variable}` from the interpolated string avoids the issue, but it means that interpolated strings cannot be used in TypeWriter output at all.

**Comment by DaveOMac** [comment]
+1

**Comment by frhagn** [comment]
Resolved in 1.8.0
